src/libs/logos.js

```diff
@@ -23,15 +23,15 @@ import { isAnvil, isBaseline, isBioDataCatalyst, isDatastage, isElwazi, isFirecl
 import * as Utils from 'src/libs/utils'
 
 
-export const getAppName = (longName = false) => Utils.cond(
+export const getAppName = ({ longName = false, capitalInitial = false } = { longName: false, capitalInitial: false }) => Utils.cond(
```

Comment:
The extensive defaulting is so a good number of call sites using the default values could remain concise.

Reply:
I thought I had run into an issue with that but don't seem to be able to reproduce it. 
Changed in [665699a](https://github.com/DataBiosphere/terra-ui/commit/665699a31b5b949bc14817ea23100b2c9ca9b01c).